### DISCUSSION TITLE
Reports on Pipeline usage for review and future discussion

### DISCUSSION THREAD

**Discussion by SarahMorleyWilkins** [daisy/pipeline#682] [General]
Dear friends, 

I presented the headline findings from the Pipeline Usage surveys at our last technical meetings, and I have attached the two reports, covering all four surveys for your perusal.
 
After my initial conversations with Flavia and Arne, Avneesh, Bert and Romain, there are perhaps four key issues for us to resolve together, including:
 
1. Our communication approach with stakeholders – how we maintain dialog, requirements, feedback and engagement.
2. How we handle and prioritize bugs, issues and feature requests.
3. How we plan how to make our development cycle more agile and responsive whilst also delivering the goals set out in the high-level operational plan.
4. Reviewing resources needed for Pipeline – firstly, identifying where know-how exists both within and outside our organizations, and secondly, identifying possible financing models.

We would like to progress these issues with you all at our future technical calls, so we can investigate and develop collaborative solutions over the coming months.

Best regards,
Sarah
 
[Findings_from_DAISY_Pipeline_Usage_Survey_1_of_Board_Members_20.10.2022.docx](https://github.com/daisy/pipeline/files/11515709/Findings_from_DAISY_Pipeline_Usage_Survey_1_of_Board_Members_20.10.2022.docx)

[Findings_from_DAISY_Pipeline_Usage_Surveys_2,3,4-02.05.2023.docx](https://github.com/daisy/pipeline/files/11515712/Findings_from_DAISY_Pipeline_Usage_Surveys_2.3.4-02.05.2023.docx)

**Comment by bertfrees** [comment]
For further refinement and discussion of the responses, I created these new threads:

- https://github.com/daisy/pipeline/discussions/705
- https://github.com/daisy/pipeline/discussions/706
- https://github.com/daisy/pipeline/discussions/707
- https://github.com/daisy/pipeline/discussions/708
- https://github.com/daisy/pipeline/discussions/709
- https://github.com/daisy/pipeline/discussions/710
- https://github.com/daisy/pipeline/discussions/711
- https://github.com/daisy/pipeline/discussions/712
- https://github.com/daisy/pipeline/discussions/713
- https://github.com/daisy/pipeline/discussions/714
- https://github.com/daisy/pipeline/discussions/715
- https://github.com/daisy/pipeline/discussions/716
